cairo/tests/ef_tests/cancun/test_state_transition.py

```diff
@@ -101,5 +108,7 @@ def test_general_state_tests(test_case: Dict) -> None:
     fetch_cancun_tests(test_dir),
     ids=idfn,
 )
-def test_execution_specs_generated_tests(test_case: Dict) -> None:
-    run_cancun_blockchain_st_tests(test_case)
+def test_execution_specs_generated_tests(
+    test_case: Dict, cairo_filepath, cairo_state_transition
```

Comment:
cairo_filepath not used

Reply:
it's used in the test collection mechanism to resolve the filepath of the cairo file to compile

Reply:
Oh right. This is not straightforward when reading the test.
Shouldn't it be a configuration issue instead of a fixture
maybe use pytest markers instead ?
-> define a custom marker in conftest 
-> add marker on your test 
-> get_closest_marker in resolve_cairo_file to retrieve the path

```
1. conftest.
config.addinivalue_line(
        "markers",
        "cairo_file(path): mark test with cairo source file path to compile"
    )
    
2. test
@pytest.mark.cairo_file(Path("cairo/ethereum_rlp/rlp.cairo"))

3. resolve_cairo_file
        marker = item.get_closest_marker("cairo_file")
```

Reply:
good idea!